packages.windows/adminrouter/setupinfo.json

```diff
@@ -0,0 +1,10 @@
+{

```

Comment:
Where are these `setupinfo.json` scripts used/referenced?

If the intention is that these scripts are called when packages are activated on a cluster node, that's probably not going to work. Pkgpanda packages must contain all the artifacts that they'll need at runtime. Installing a package should mean nothing more than extracting the package tarball, symlinking its files into the filesystem at the install root, copying its systemd units to `/etc/systemd/system/`, and setting up host-level users/groups. Also these setup scripts would need to be idempotent, because a package may be re-activated during an upgrade, meaning (I assume) we would-rerun its setup.